front-end/src/app/contacts/contacts-table/contacts-table.component.ts

```diff
@@ -299,7 +299,7 @@ export class ContactsTableComponent implements OnInit, OnDestroy {
         }
 
         this._contactsService.addUIFileds(res);
-        this._contactsService.mockApplyFilters(res, this.filters);
+        // this._contactsService.mockApplyFilters(res, this.filters);
```

Comment:
Is it worth keeping this line as a comment?

Reply:
Yes, for now as the filter code that is not working is in that call so we know where the call was made later on.